comms/discovery/schema/schema.ts

```diff
@@ -155,9 +156,9 @@ export enum ChatPermission {
    */
   ALL = 'all',
   /**
-   * Messages are only allowed for users that have tipped me
+   * Messages are only allowed for users that have tipped me OR users I follow
    */
-  TIPPERS = 'tippers',
+  TIPPERS_OR_FOLLOWEES = 'tippersOrFollowees',
```

Comment:
sorry this is my fault this got lost in communication - we adopted the other part of the proposal but not this part, we're keeping tip gated and followee gated separate for now

Reply:
changed back to tippers only